CloudBoltAutomation/Functions/Get-CBGroup.ps1

```diff
@@ -29,16 +29,17 @@ function Get-CBGroup
     param(
         [Parameter(Mandatory)]
         [object]
-        # The session/connecton to use. Use `New-CBSession` to create a session object.
+        # The session/connecton to the CloudBolt instance to use. Use `New-CBSession` to create a session object.
         $Session,
 
         [Parameter(Mandatory,ParameterSetName='SpecificGroup',ValueFromPipeline,ValueFromPipelineByPropertyName)]
         [int]
+        # The ID of the group to get.
         $ID,
 
         [Parameter(ParameterSetName='AllGroups')]
         [int]
-        # How many results should each request return? The function will do the paging for you and return all groups. The default page size is an obscenely high number.
+        # The page size. CloudBolt pages all lists of objects. The `Get-CBEnvironment` function will make an HTTP request for every page of results so that all environments get returned. The default page size is 10.

```

Comment:
Curious about how the `$PageSize` parameter works, since the doc says the default is 10 but we don't set a default.

With this parameter being an `[int]` it's going to default to `0` if it's not specified. Does the CloudBolt server then default the page size to 10 if the page size that's given in the request is 0?